### ISSUE TITLE
Instructions in README.md are insufficient to get stt_whisper to work

### ISSUE THREAD

**Issue by schniepp** [nextcloud/stt_whisper#22]
Running most recent NC version here (27.1, "hub 6") here. I installed the stt_whisper app from app store, then installed the medium language model, which appeared to go successfully. Also installed the "OpenAI and LocalAI integration " (integration_openai), which is supposed to allow me using whisper for dictation. However, I cannot get it to work. 

The README.md does not provide instructions regarding which NC settings have to be used to make this work. In particular, it is not clear which URL/port needs to be used in the NC "Connected Accounts" administrator settings. It appears that there are also administration settings to be chosen in the "Artificial Intelligence" menu, but it is not clear which ones need to be used to make stt_whisper work. Trial and error has not resulted in a viable solution. In particular, it is not clear whether the stt_whisper app will listen at a particular port/URL.

**Comment by muchachagrande** [comment]
Hi @schniepp. In a previous version of stt_whisper the procedure you described was enough to make whisper work with Talk Recording. This app is, for now, intended to do text transcriptions on Nextcloud Talk video recordings.
However, since some update, could not determine which, it stopped from working on my installation, so your problem may be not related to a miss configuration but to some recent issue.
May even be related to some Nextcloud change and not to the stt_whisper itself.

**Comment by muchachagrande** [comment]
Hi again. I was digging into my configuration problem and I found that it was added a configuration switch to enable automatic recording transcriptions.
Without this switch set to 'yes' the transcription will not happen.

occ config:app:set spreed call_recording_transcription --value yes

This switch must have been added recently because I had stt_whisper working some time ago and it stopped working after some update.
